### PR TITLE
SPCTR-2288-code-refactor-of blocks

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description
Removal of get_fallback function calls from frontend.css.php of blocks forms, gf-styler, google-map, how-to, icon-list, image, image-gallery.
### Screenshots
forms: https://d.pr/v/Rmi8Kz
gf-styler: https://d.pr/v/jEN7Od
google-map: https://d.pr/v/kjRB3q
how-to: https://d.pr/v/frVzm0
icon-list: https://d.pr/v/PMMchX
image: https://d.pr/v/d6J3my
image-gallery: https://d.pr/v/oGW7LH

### How has this been tested?
- have added null value to the attributes in editor.
- using x-debug have checked the value of the attributes it take default values if we give null.
- by giving a value to the attributes it take the current value we have passed.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: includes/blocks/how-to/frontend.css.php [hunk 1]
```diff
@@ -12,11 +12,7 @@
 
 $block_name = 'how-to';
 
-$row_gap_fallback  = UAGB_Block_Helper::get_fallback_number( $attr['row_gap'], 'row_gap', $block_name );
-$trow_gap_fallback = UAGB_Block_Helper::get_fallback_number( $attr['rowGapTablet'], 'rowGapTablet', $block_name );
-$mrow_gap_fallback = UAGB_Block_Helper::get_fallback_number( $attr['rowGapMobile'], 'rowGapMobile', $block_name );
-
-$step_gap_fallback = UAGB_Block_Helper::get_fallback_number( $attr['step_gap'], 'step_gap', $block_name );
+$step_gap_fallback = $attr['step_gap'];
```
Comment (bsf-zanev) [inline includes/blocks/how-to/frontend.css.php:15]: We don't need this variable.

File: includes/blocks/how-to/frontend.css.php [hunk 2]
```diff
@@ -12,11 +12,7 @@
 
 $block_name = 'how-to';
```
Comment (bsf-zanev) [inline includes/blocks/how-to/frontend.css.php:13]: Is the Block Name being used anywhere else?

**Review by bsf-zanev — CHANGES_REQUESTED** [review]
Please address the comments below.

File: includes/blocks/how-to/frontend.css.php (hunk 1 above)
Comment (rajmohanreddy774) [reply includes/blocks/how-to/frontend.css.php]: removed

File: includes/blocks/how-to/frontend.css.php (hunk 2 above)
Comment (rajmohanreddy774) [reply includes/blocks/how-to/frontend.css.php]: removed

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

**Review by rahulvarma722 — APPROVED** [review]

**Pull request merged**